### PR TITLE
add libwebp/1.1.0

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **libwebp/1.1.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libwebp/all/conanfile.py [hunk 1]
```diff
@@ -4,87 +4,95 @@
...
-        return cmake
+        return self._cmake
 
     def build(self):
-        tools.patch(base_path=self._source_subfolder, patch_file='0001-fix-dll-export.patch')
-        tools.patch(base_path=self._source_subfolder, patch_file='0002-enable-webpmux.patch')
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        # build libwebpmux (not in patch file to avoid one patch per version)
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
```
Comment (uilianries) [inline recipes/libwebp/all/conanfile.py]: Why did you remove that option and is forcing it now? As optional from author project, you should give the same proposal, do not force an option.

Comment (SpaceIm) [reply recipes/libwebp/all/conanfile.py]: Which option?
The removed patch added an option in CMakeLists unused in conanfile and any file in libwebp (before or after patch), and removed an option set anyway to false by conanfile. The others modifications of this patch had the same effect than the simple replace_in_file proposed here.

Comment (uilianries) [reply recipes/libwebp/all/conanfile.py]: > Which option?

WEBP_BUILD_WWEBP

> The removed patch added an option in CMakeLists unused in conanfile and any file in libwebp (before or after patch), and removed an option set anyway to false by conanfile.

I see, so was WWEBP optional by the author or not?

Comment (SpaceIm) [reply recipes/libwebp/all/conanfile.py]: It was useless, a typo I think or something which was used before 1.0.3. Take a look to 1.0.3 source code of libwebp and to the old conan recipe.
